src/qibolab/qubits.py

```diff
@@ -40,6 +41,23 @@ def channels(self) -> list[ChannelId]:
             if x is not None
         ]
 
+    @classmethod
+    def default(cls, name: QubitId, channels: Optional[list[str]] = None, **kwargs):
+        """Create a qubit with default channel names.
+
+        Default channel names follow the convention:
+        '{qubit_name}/{channel_type}'
+
+        Args:
+            name: Name for the qubit to be used for channel ids.
+            channels: List of channels to add to the qubit.
+                If ``None`` the following channels will be added:
+                probe, acquisition, drive and flux.
+        """
+        if channels is None:
+            channels = ["probe", "acquisition", "drive", "flux"]
```

Comment:
Almost irrelevant: to avoid name duplication, we could add metadata to the fields, and iterate them to find which are the default ones.

Reply:
Done in b4e58161df2974b87e2fab73c1250411e3017af3. If there is a better way to do it, let me know, or feel free to do it.